packages/ecc-utils-design/src/components/form/form.ts

```diff
@@ -517,6 +588,27 @@ export default class EccUtilsDesignForm extends LitElement {
     this.formState = "idle";
   }
 
+  private handleSubmit(e: Event) {
+    e.preventDefault();
+    const form = this.shadowRoot?.querySelector("form");
+    const isValid = form?.reportValidity();
+    if (!isValid) {
+      return;
+    }
+    if (Object.keys(this.form).length === 0) {
+      this.error({ message: "Form is empty" });
+      return;
+    }
```

Comment:
Why is this added?

Reply:
it's supposed to prevent empty forms from being submitted.  I just felt it was a basic check we needed. Should I remove it?